__Multivariable Calculus - Fourth Edition</br>
by James Stewart </br>
Chapter 16 - Multiple Integrals__

In this chapter, we extend the idea of a definite integral to double and triple integrals of functions of two or three variables.  These ideas are then used to compute volumes, surface areas, masses and centroids of more general regions than we were able to consider in Chapter 6 and 9.  We also use double to integrals to calculate probalities when two random variables are involved.

__16.1 Double Integrals over Rectangles__ 

In much the same way that our attempt to solve the area problem led to the definition of a definite integral, we now seek to find the volume of a solid and in the process we arrive at the definition of a double integral.

__Review of the Definite Integral__

First let's recall the basic facts concerning definite integrals of functions of a single variable.  If $f(x)$ is defined for $a≤x≤b$, we start by dividing the interval $[a,b]$ into n sub-intervals $[x_{i-1},x_{i}]$ of equal width $x = (b-a)/n$ and we choose sample points $x_{i}$ in these subintervals.  Then we form the Riemann sum

$$ \Sigma_{i=1}^n f(x_{i})Δx$$

and take the limit of such sums as $n→∞_{love}$ to obtain the definite integral of $f$ from $a$ to $b$:

$$ \int_a^b f(x)dx = \lim\limits_{n \to \infty}\Sigma_{i=1}^n f(x_{i})Δx$$

In the special case where $f(x)≥0$, the Riemann sum can be intepreted as the sum of the areas of the approximating rectangles, and  $\int_a^b f(x)dx $ represents the area under the curve $y = f(x)$ from $a$ to $b$.

__Volumes and Double Integrals__

In a similar manner we consider a function f of two variables defined on a closed rectangle

$ R = [a, b] × [c, d] = \{{(x,y)∈R^{2}|a≤x≤b, c≤y≤d}\}$

and we first suppose that $f(x,y)≥0$.  The graph of f is a surface with equation $z = f(x,y)$.  Let S be the solid that lies above $R$ and under the graph of $f$, that is,


$ S = \{ (x,y,z) ∈R^{3}  |0≤z≤f(x,y), (x,y)∈R $ }

(See Figure 2.) Our goal is to find the volume of S.

The first step is to divide the rectangle R into subrectangles. We accomplish this by dividing the interval $[a,b]$ into m subintervals $[x_{i-1},x_{i}]$ of equal width $Δx = (b-a)/m$ and dividing $[c,d]$ into n subintervals $[y_{j-1}, y_{j}]$ of equal width $Δy = (d-c)/n$.  By drawing lines parallel to the coordinate axes through the endpoints of these subintervals as in Figure 3, we form the subrectangles </br>

$ R_{ij} = [x_{i-1},x_{i}]×[y_{j-1}, y_{j}] = \{ (x,y)|x_{i-1}≤x≤x_{i} ,y_{j-1}, y_{j} \} $

__The Midpoint Rule__

The methods that we used for approximating single integrals (the Midpoint Rule, the Trapezoidal Rule, Simpson's Rule) all have counterparts for double integrals.  Here we consider only the Midpoint Rule for double integrals.  This means that we use a double Riemann sum to approximate the double integral, where the sample point $(\bar x_{ij}, \bar y_{ij})$ in $R_{ij}$ is chosen to be the center $(x_{i},y_{j})$ of $R_{ij}$.  In other words, $ \bar x_{i}$, is the midpoint of $[x_{i-1}, x_{i}]$ and $ \bar y_{j}$ is the midpoint of $[y_{j-1}, y_{j}]$.

$ \iint_R f(x,y)dA ≈ \Sigma_{i=1}^m {i}\Sigma_{j=1}^n f(x_{i}, y_{j})ΔA $

where $ \bar x_{i}$ is the midpoint of $[x_{i-1}, x_{i}]$ and $ \bar y_{j} $ is the midpoint of $[y_{i-1}, y_{i}]$.


__Example 3__

Use the Midpoint Rule with $m = n = 2$ to estimate the value of the integral $ \iint_R (x - 3y^{2})dA$ , where $ R = .

__Solution__

In using the Midpoint Rule with $m = n = 2$, we evaluate at the centers of the four subrectangles shown in Figure 10. so $ \bar s_{1} = $


$\iint_{R} f(x,y)dA ≈ \Sigma_{i=1}^m {i}\Sigma_{j=1}^n f(\bar x_{i}, \bar y_{j})ΔA$

$ = f(\bar x_{1},\bar y_{1})ΔA + f(\bar x_{1},\bar y_{2})ΔA + f(\bar x_{2},\bar y_{1})ΔA + f(\bar x_{2},\bar y_{2})ΔA$

__Average Value__

Recall from Section 6.5 that the average value of a function f of one variable deifed on an interval $[a,b]$ is

$ f_{ave} = \frac{1}{b-a} \int_a^b f(x)dx $

In a similar fashion we define the __average value__ of a function f of two variables defined on a rectangle R to be 

$ f_{ave} = \frac{1}{A(R)} \iint_a^b f(x,y)dA $

where $A(R)$ is the area of $R$.
If $f(x,y)≥0$, the equation

$A(R) \times f_{ave} \iint_{R} f(x,y)dA $

says that the box with base $R$ and height $f_{ave}$ has the same volume as the solid that lies under the graph of $f$.  \[ If $z=f(x,y)$ describes a mountainous region and you chop off the tops of the mountains at height $f_{ave}$, then you can use them to fill in the valleys so tha the region becomes completely flat.]




__Exercise 1.__
Find approximations to $\iint_{R} (x-3y^{2})dA $ using the same subrectangles as in Example 3 but choosing the sample point to be the (a) upper left corner, (b) upper right corner, (c)lower left corner, (d) lower right corner of each subrectangle.

In [1]:
def solve_ex1(x,y):
    f = x-3*(y*y)
    print (f)

__(a)__ let $ \bar x_{1} = 0$, $\bar x_{2} = 1$, $\bar y_{1} = \frac{3}{2}$, and $\bar y_{2} = 2$.  The area of each subrectangle is still $ ΔA= \frac{1}{2}. $

$\iint_{R} (x-3y^{2})dA ≈ \Sigma_{i=1}^m {i}\Sigma_{j=1}^n f(x_{i}, y_{j})ΔA$

$ = f(\bar x_{1},\bar y_{1})ΔA + f(\bar x_{1},\bar y_{2})ΔA + f(\bar x_{2},\bar y_{1})ΔA + f(\bar x_{2},\bar y_{2})ΔA$

$ = f(0,\frac{3}{2})ΔA + f(0,2)ΔAf + f(1,\frac{3}{2})ΔA + f(1,2)ΔA $

In [2]:
solve_ex1(0, 1.5)
solve_ex1(0, 2)
solve_ex1(1, 1.5)
solve_ex1(1, 2)

-6.75
-12
-5.75
-11


$ = (-6.75)(0.5) + (-12)(0.5) + (-5.75)(0.5) + (-11)(0.5) $

$ = -17.75 $


__(b)__ let $ \bar x_{1} = 1$, $\bar x_{2} = 2$, $\bar y_{1} = \frac{3}{2}$, and $\bar y_{2} = 2$. The area of each subrectangle is still $ ΔA= \frac{1}{2}. $

$\iint_{R} (x-3y^{2})dA ≈ \Sigma_{i=1}^m {i}\Sigma_{j=1}^n f(x_{i}, y_{j})ΔA$

$ = f(\bar x_{1},\bar y_{1})ΔA + f(\bar x_{1},\bar y_{2})ΔA + f(\bar x_{2},\bar y_{1})ΔA + f(\bar x_{2},\bar y_{2})ΔA$


In [3]:
solve_ex1(1, 1.5)
solve_ex1(1, 2)
solve_ex1(2, 1.5)
solve_ex1(2, 2)

-5.75
-11
-4.75
-10


$ = (-5.75)(0.5) + (-11)(0.5) + (-4.75)(0.5) + (-10)(0.5) $

$ = 15.75$

__(c)__ let $ \bar x_{1} = 0$, $\bar x_{2} = 1$, $\bar y_{1} = 1$, and $\bar y_{2} = \frac{3}{2}$.  The area of each subrectangle is still $ ΔA= \frac{1}{2}. $

$\iint_{R} (x-3y^{2})dA ≈ \Sigma_{i=1}^m {i}\Sigma_{j=1}^n f(x_{i}, y_{j})ΔA$

$ = f(\bar x_{1},\bar y_{1})ΔA + f(\bar x_{1},\bar y_{2})ΔA + f(\bar x_{2},\bar y_{1})ΔA + f(\bar x_{2},\bar y_{2})ΔA$

In [4]:
solve_ex1(0, 1)
solve_ex1(0, 1.5)
solve_ex1(1, 1)
solve_ex1(1, 1.5)

-3
-6.75
-2
-5.75


$ = (-3)(0.5) + (-6.75)(0.5) + (-2)(0.5) + (-5.75)(0.5) $

$ =  -8.75$

__(d)__ let $ \bar x_{1} = 1$, $\bar x_{2} = 2$, $\bar y_{1} = 1$, and $\bar y_{2} = \frac{3}{2}$. The area of each subrectangle is still $ ΔA= \frac{1}{2}. $

$\iint_{R} (x-3y^{2})dA ≈ \Sigma_{i=1}^m {i}\Sigma_{j=1}^n f(x_{i}, y_{j})ΔA$

$ = f(\bar x_{1},\bar y_{1})ΔA + f(\bar x_{1},\bar y_{2})ΔA + f(\bar x_{2},\bar y_{1})ΔA + f(\bar x_{2},\bar y_{2})ΔA$


In [5]:
solve_ex1(1, 1)
solve_ex1(1, 1.5)
solve_ex1(2, 1)
solve_ex1(2, 1.5)

-2
-5.75
-1
-4.75


$ = (-2)(0.5) + (-5.75)(0.5) + (-1)(0.5) + (-4.75)(0.5) $

$ = -6.75 $

__Exercise 2__ Find the approximation to the volume in example 1 if the Midpoint Rule is used

__16.2__ Iterated Integrals

__Fubini's Theorem__
if $f$ is continuous on the rectangle R = {(x,y)|axb, cyd}, then

$\iint_{R} f(x,y)dA = \int_a^b \int_c^d f(x,y)dydx =  \int_c^d \int_a^b f(x,y)dxdy  $    

The proof for Fubini's theorem is too difficult to include in this book, but we can at least give an intuitive indication of why it is true for the case where f(x,y) 0.  Recall that $f$ is positive, then we can interpret the double integral as the Volume V on the solid Sthat lies above R and under the surface z=f(x,y).  But we have another formula that we used for volume in Chapter 6, namely, 

$ V + \int_a^b A(x)dx $

__Exercise 3__
$\int_1^3 \int_0^1 (1+4xy)dxdy = \int_1^3 [x + \frac{4}{2}x^2y]_0^1 dy = \int_1^3 ([1+2y]-[0])dy = \int_1^3 (1+2y) dy$
$ = [y+ \frac{2}{2}y^2]_1^3$
$=12-2= 10$

__Exercise 4__
$\int_2^4 \int_{-2}^4 (x^2+y^2)dydx = \int_2^4 [x^2y + \frac{1}{3}y^3]_{-1}^1dx = \int_2^4 ([x^2+\frac{1}{3}]-[(-x^2-\frac{1}{3}])dx=\int_2^4 2x^2 + \frac{2}{3}dx = 2\int_2^4 x^2 + \frac{1}{3} = 2[\frac{1}{3}x^3 +\frac{1}{3}x]_2^4 = 2([\frac{4^3}{3} + \frac{4}{3}] - [\frac{2^3}{3}])= 2(\frac{4^3-2^3}{3} + \frac{2}{3})= \frac{116}{3}$

__Exercise 5__</br>
$\int_0^{π/2} \int_0^{π/2} sinxcosy dydx = \int_0^{π/2} sinx (\int_0^{π/2} cosy dy)dx = \int_0^{π/2}[siny]_0^{π/2} dx = \int_0^{π/2} sinx([sin(π/2)-sin(0)])dx = \int_0^{2π}sinx = [-cosx]_0^{π/2}=[0]-[-1]=1$

In [ ]:
import matplotlib.pyplot as plt
import numpy as np

ax = plt.axes(projection='3d')

x = np.linspace(0, 1, 1000);
y = np.linspace(0, 1, 1000);
z = (4-x-2);

ax.mesh3D(x, y, z, 'black')
plt.title('Volume of Integration');



__16.2 Exercises__ p.1014


__13__

$\int \int_R (6x^2y^3 - 5y^4)dA$
$R=\{(x,y)|0≤x≤3, 0≤y≤1\}$

$= \int_1^3 \int_0^3 6x^2y^3-5y^4 dxdy$
$=\int_0^1 [\frac{6}{3}x^3y^3 -5y^4x]_0^3dy$
$=\int_0^1 ([2(3^3)y^3 - 5y^4(3)] - [0]) dy$
$\int_0^1 ([(2)(27)y^3 -5y^4(3)] - [0]) dy$
$= \int_0^1 2(27)(y^3) - 15y^4 dy$
$= [\frac{2(27)}{4}y^4-\frac{15}{5}y^5]_0^1$
$= [\frac{27}{2}y^4-3y^5]_0^1 = [\frac{27}{2}]-[0]= \frac{27-6}{2} = \frac{21}{2}=10.5$


__Exercise 14__</br>
$\int \int_R xye^y dA$
$R = {(x,y)|0≤x≤2, 0≤y≤1}$

$\int_0^1 \int_0^2 xye^y dxdy = \int_0^1 [\frac{1}{2}x^2ye^y]_0^2 dy$
$= \int_0^1 ([2ye^y]-[0])dy = 3\int_0^1 ye^y dy$</br>
Integration by parts</br>
$let u = y$</br>
$du = dy$</br>
$dv = e^x$</br>
$v= \int dv = \int e^y dy = e^y$</br>
$ye^y dy = ye^y - \int e^y dy = ye^{y}-e^{y} = e^y(y-1)$</br>
$[e^y(y-1)]_0^1 = [e^1(0)]-[e^0(-1)] = e^0 = 1$

__Exercise 15__

$\int \int_R \frac{xy^2}{x^2 + 1}$
$R = {(x,y)|0≤x≤1, -3≤y≤3}$

$\int_0^1 \int_{-3}^{3} y^2 \frac{x}{x^2+1} dydx$
$= \int_0^1 [\frac{y^3}{3}\frac{x}{x^2+1}]_{-3}^{3} dx = \frac{9x}{x^2+1} - \frac{-9x}{x^2+1}$

$=9 \int_0^1 \frac{2x}{x^2+1}= 9\int_0^1 2x (x^2+1)^{-1} dx$
$let u = (x^2+1)$
$let du= 2x$

$\int_0^1 2x(x^2-1)^{-1}dx = 9[ln|(x^2+1)|]_0^1 = 9[ln2-ln1] = 9ln|2|$




__16.3 Double Integrals Over General Regions__



For single integrals, the region over which we integrate is always an interval.  But for double integrals, we want to be able to integrate a function $f$ not just over rectangles but also over regions $D$ of more general shape, such as the one illustrated in Figure 1.  We suppose that $D$ is the bounded region, which means that $D$ can be enclosed in a rectangular region.  Then we define a new function $F$ with domain $R$ by 

$ F(x,y) = \Bigl\{_{f(x,y)\;if\;(x,y)\;is\;in\;D}^{0\;if\;(x,y)\;is\;in\;R\;but\;not\;in\;D} $

If the double integral of $F$ exists over $R$, then we define the double integral of $f$ over $D$ by

$ \int \int_D f(x,y)dA = \int \int F(x,y)\;dA $, where $F$ is given by the domain defined above.

This sense because $R$ is a rectangle and so $\int \int_R F(x,y)\;dA$ has been previously defined.  The procedure we have used is reasonable because the values of $F(x,y)$ are 0 when $(x,y)$ lies outisde D and so they contribute nothing to the integral.  This means that it doesn't matter what rectangle $R$ we use as long as it contains $D$.</br>
</br>
In the case where $f(x,y)≤0$, we can still interpret $\int\int_D f(x,y)\;dA $ as the volume of the solid that lies above $D$ and and under the surface $z = f(x,y)$ (the graph of $f$).  You can see that this is reasonable by comparing the graphs of $f$ and $F$ and remembering that $\int \int_R F(x,y)\;dA $ is the volume under the graph of $F$. 
</br>
F is likely to have discontinuities at the boundary points of D.  Nonetheless, if f is continuous on D and the boundary curve of D is "well behaved" then it can be shown that $\int \int_R F(x,y)\; dA $ exists and therefore $\int \int_D f(x,y)\;dA$ exists.  In particular, this is the case for the following types of regions:
</br>
A plane region D is said to be of __type 1__ if it lies between the graphs of two continuous functions of x, that is,</br>
</br>
$D = {(x,y)|\;a≤x≤b, g_1(x)≤y≤g_2(x)}$
</br>
where $g_1$ and $g_2$ are continuous on [a,b]

...

We also consider plane regions of __type II__, which can be expressed as</br>
$D = \{(x,y)|\;c≤y≤d, h_1(y)≤x≤h_2(y)\}$</br>
where $h_1$ and $h_2$ are continous.  Using the same methods as earlier, we can show that </br>
$\int \int_D f(x,y)\;dA$.



__19.__ Use polar coordinates to find the volume of the given solid.
Under the parabaloid $z = x^2 + y^2$ and above the disk $x^2 + y^2≤9$. 

$ V = \int \int_D (x^2 + y^2) dA $ </br>
$x = rsin\theta$</br>
$y = rcos\theta$</br>

$ \int_0^{2\pi} \int_0^3 (x^2 + y^2)\;drd\theta $</br>
$ = \int_0^{2\pi} \int_0^3  (rsin\theta)^2 + (rcos\theta)^2 r\;drd\theta $</br>
$ = \int_0^{2\pi} \int_0^3  r^3sin^3theta + r^3cos^2\theta \;drd\theta  $</br>
$ = \int_0^{2\pi} [\frac{1}{4}r^4sin^2\theta + \frac{1}{4}r^4cos^2\theta]_0^3 \;d\theta $</br>
$ = \int_0^{2\pi} ([\frac{1}{4}(81)sin^2\theta + \frac{1}{4}(81)cos^2\theta] - [0]) \;d\theta $</br>
$ = \int_0^{2\pi} ([\frac{81}{4}sin^2\theta + \frac{81}{4}cos^2\theta]) \;d\theta $</br>
Using the pythagorean theorem $sin^2\theta + cos^2\theta = 1$
$ = \frac{1}{4} \int_0^{2\pi} (81) \;d\theta $</br>
$ = \frac{1}{4} [81\theta]_0^{2\pi} $</br>
$ = \frac{1}{4} ((81)(2\pi)) = \frac{81\pi}{2} $ (this is the book answer)


__20.__ Inside the sphere $x^2 + y^2 = z^2 = 16 $ and outside the cylinder $x^2 + y^2 = 4$

$Volume\;of\;Sphere = \frac{4}{3}\pi r^3$

__16.4 Double Integrals In Polar Coordinates__

Density and Mass

In Section 9.3, we were able to use single integrals to compute moments and the center of mass of a thing plate or lamina with constant density.  But now, equipped with the double integral, we can consider a lamina with variable density.  Suppose the lamina occupies a region $D$ of the xy-plane and its density (in units of mass per unit area) at a point $(x,y)$ in D is given by $ρ(x,y)$, where ρ is a continuous function of $D$.  This means that 

$ ρ(x,y) = lim \frac{Δm}{ΔA}$

__Change to Polar Coordinates in a Double Integral__</br>
If $f$ is continuous on a polar rectangle $R$ given by $0≤a≤r≤b, 0≤β-α≤2\pi$, then</br>
$ \int \int_R f(x,y) dA = \int_α^β \int_a^b$$ f (r cos \theta, r sin \theta)r dr d\theta $

__Example 1__ Evaluate $\int \int_R (3x+4y^2)dA$, where $R$ is the region in the upper half-plane bounded by the circles $x^2 + y^2 =1$ and $x^2 + y^2 = 4$.</br>
SOLUTION The region can be described as 
$ R = \{(x,y)|y≤0, 1≤x^2 + y^2≤4 \} $</br>
It is the half-ring shown on p. 1023, and in polar coordinates it is given by $1≤r≤2$, $0≤\theta≤\pi$. Therefore

$\int \int_R (3x + 4y^2)\;dA = \int $

__Exercises 1-6__ 
A region R is shown.  Decide whether to use polar coordinates or rectangular coordinates and write $\int \int_R F(x,y) dA$ as an iterated integral, where $f$ is an arbitrary continuous function on $R$.

__1.__ For a circle, polar coordinates(r,\theta) will be easier to describe and integrate over

$ R = \{(r,\theta)|0≤r≤2, 0≤\theta≤2\pi\}$</br></br>
equation of circle in book $ x^2 + y^2 = r^2 $ or $ x^2 + y^2 = 4 $, where $x = rcos\theta$ and $y = rsin\theta$</br></br>
$ \int_0^{2\pi} \int_0^2 f(rcos\theta, rsin\theta) rdr\theta $

__2.__ 
For triangle region in x-y plane with points (0,0), (2,0) & (0,2). 

$ \int_0^2 \int_0^2 (-x + 2)\;dxdy $ (book answer is  $ \int_0^2 \int_0^2 f(x,y) \;dxdy $

__3.__ For triangle region in x-y plane with points (2,2), (-2,-2), and (-2, 2).

$ \int_-2^2 \int_0^2 (2-x) \;dxdy $

__4.__ For disk region in x-y plane bounded by circle of radius 1 and circle of radius 3 in the first quadrant.

$ \int_0^\frac{\pi}{2} \int_0^3   f(rcos\theta, rsin\theta)\;drd\theta $ (book answer)

__Exercises 15-18__  Use a double integral to find the area of the region

__15.__ One loop of the rose $ r = cos 3\theta$ 

Using example 3 as a guide: 
$D = {(r,\theta)|\frac{-\pi}{4}≤\theta≤\frac{\pi}{4}, 0≤r≤cos3\theta}$
</br>
$\int_{\frac{-\pi}{4}}^{\frac{\pi}{4}} \int_0^{cos(3\theta)} r\;dr d\theta $
$= \int_{\frac{-\pi}{4}}^{\frac{\pi}{4}} [\frac{1}{2} r^2]_0^{cos(3\theta)}\;dr d\theta $
$= \int_{\frac{-\pi}{4}}^{\frac{\pi}{4}} [\frac{1}{2} cos^23\theta] - [0] \; d\theta $
$= \frac{1}{2} \int_{\frac{-\pi}{4}}^{\frac{\pi}{4}} cos^23\theta \; d\theta $

From Table of Integrals</br>
$\int cos^2u\;du = \frac{1}{2}u + \frac{1}{4}sin2u $

$= (\frac{1}{2})(\frac{1}{3}) [\frac{3\theta}{2} + \frac{1}{4} sin6\theta ]_{\frac{-\pi}{4}}^{\frac{\pi}{4}}$

$= (\frac{1}{6}) ([\frac{3(\frac{\pi}{4})}{2} + \frac{1}{4} sin6(\frac{\pi}{4})] -         [\frac{3(\frac{-\pi}{4})}{2} + \frac{1}{4} sin6(\frac{-\pi}{4}) ]) $

$= (\frac{1}{6}) ([ \frac{6\pi}{4} + \frac{1}{4} sin(\frac{3\pi}{2})- \frac{-6\pi}{4} - \frac{1}{4} sin(\frac{-3\pi}{2}) ]) $


$= \frac{1}{24}[12\pi -1 - 1] = -\frac{1}{12}$, book answer = $\frac {\pi}{12}$

In [ ]:
__Exercises 27-30__ Evaluate the 

__16.5 Application of Double Integrals__

__Exercises 1-6__ Evaluate the iterated integral

__1.__   $\int_0^1 \int_0^{x^{2}} (x+2y)dydx$

$ \int_0^{x^{2}} (x+2y)dy =[ xy + y^2  ]_{y=0}^{y = x^{2}}  =$ $ [x(x^2) + (x^2)^2] - [0 + 0] = x^3 + x^4$ 

$\int_0^1  (x^3 + x^4)\;dx = [ \frac{1}{4} x^{4} + \frac{1}{5} x^{5}]_0^1 = [\frac{1}{3} y] - [0] = \frac{9}{20}$

__2.__   $\int_1^2 \int_y^2 (xy)\;dxdy$

$ \int_y^2 (xy)\;dx = [ x^2y ]_{x=y}^{x=2} = [(2^2)y] - [(y^2)y ] = 4y - y^3 $

 $\int_1^2  4y - y^3 dy  = [ 2y^2 - \frac{y^4}{4} ]_1^2 = [8-4] - [2-\frac{1}{4}] = 4 - \frac{7}{4} = \frac{9}{4}$ 

__3.__   $ \int_0^1 \int_y^{e^y} \sqrt{x} dxdy$

= $ \int_0^1 \int_y^{e^y} x^{(\frac{1}{2})} dxdy$

= $ \int_0^1 [  \frac{x^{(\frac{3}{2})}}{\frac{3}{2}} ]_{x=y}^{x={e^y}} dy$ = $ \int_0^1 [ 2\frac{2*x^{(\frac{3}{2})}}{3} ]_{x=y}^{x={e^y}} dy$

= $ \int_0^1 ([ \frac{2*(e^y)^{(\frac{3}{2})}}{3} ] - [\frac{2*(y^{(\frac{3}{2})}}{3}]) dy$ =  $ \frac{2}{3} \int_0^1 (e^y)^{(\frac{3}{2})} - y^{(\frac{3}{2})} dy$

$ = \frac{2}{3} [\frac{2}{3} (e^y)^{(\frac{3}{2})} - \frac{2}{5} y^{(\frac{5}{2})} ]_{y=0}^{y=1}$

$ = \frac{2}{3} ( [\frac{2}{3} (e^1)^{(\frac{3}{2})} - \frac{2}{5} 1^{(\frac{5}{2})} ] - [\frac{2}{3} (e^0)^{(\frac{3}{2})} - \frac{2}{5} 0^{(\frac{5}{2})} ] )$

$ = \frac{2}{3} ( [\frac{2}{3} e^{(\frac{3}{2})} - \frac{2}{5} 1^{(\frac{5}{2})} ] - [\frac{2}{3}  (1)^{(\frac{3}{2})}  - 0]) $

$ = \frac{2}{3} ( [\frac{2}{3} e^{(\frac{3}{2})} - \frac{2}{5} ] - [\frac{2}{3}] ) $

$ = \frac{2}{3} ( [\frac{2}{3} e^{(\frac{3}{2})} - \frac{6}{15} ] - [\frac{10}{15}] ) $ = $ \frac{2}{3} (\frac{2}{3} e^{(\frac{3}{2})} - \frac{16}{15}) $

= $ \frac{4}{9} e^{(\frac{3}{2})} - \frac{32}{45} $

__4.__   $ \int_0^1 \int_x^{2-x} (x^{2} - y)\;dydx$

= $ \int_0^1 [yx^{2} - \frac{y^2}{2}]_{y=x}^{y=2-x}\;dx= \int_0^1 ([(2-x)x^{2} - \frac{(2-x)^2}{2}] - [x(x^{2}) - \frac{x^2}{2}])\;dx $

= $ \int_0^1 (2x^{2} - x^3 - \frac{4 - 4x + x^2}{2} - x^{3} + \frac{x^2}{2})\;dx $ = $ \int_0^1 (\frac{4x^{2}}{2} - 2x^3 - \frac{4 - 4x + x^2}{2} + \frac{x^2}{2})\;dx $

= $ \int_0^1 (\frac{5x^{2}}{2} - 2x^3 - \frac{4 - 4x + x^2}{2})\;dx  = \frac{1}{2} \int_0^1 (5x^{2} - 4x^3 - (4 - 4x + x^2)\;dx$

= $ \frac{1}{2} \int_0^1 (5x^{2} - 4x^3 - 4 + 4x - x^2)\;dx = \frac{1}{2} \int_0^1 (-4x^3 + 4x^{2} + 4x - 4)\;dx$

 = $ \frac{1}{2} [\frac{-4x^4}{4} + \frac{4x^{3}}{3} + \frac{4x^2}{4} - 4x]_0^1 $ = $ \frac{1}{2} [-x^4 + \frac{4x^{3}}{3} + 4x^2 - 4x]_0^1 $

$ = \frac{1}{2} (-(1)^4 + \frac{4(1)^{3}}{3} + 4(1)^2 - 4(1)) - \frac{1}{2}(0) $

$ =  \frac{1}{2} (-1 + \frac{4}{3} + 4 - 4 - 0) = \frac{1}{2} (-1 + \frac{4}{3}) = \frac{1}{6}$

__5.__   $ \int_0^{\pi/2} \int_0^{cos\theta} e^{sin\theta}\;drd\theta$ 

=  $ \int_0^{\pi/2}  [re^{sin\theta}]_{r=0}^{r =cos\theta}\;d\theta$ 

=  $ \int_0^{\pi/2}  [cos\theta e^{sin\theta}] - [(0)e^{sin\theta}] \;d\theta$ = $  \int_0^{\pi/2}  cos\theta e^{sin\theta}\;d\theta $

Using integrations by parts: </br>
$\int u dv = uv - \int vdu $</br>
let $u =  e^{sin\theta}$, and $ dv = cos\theta $</br> 
then $du = e^{sin\theta}$, and $v = sin\theta $
</br>

$  \int_0^{\pi/2}  cos\theta e^{sin\theta}\;d\theta = sin\theta e^{sin\theta} - \int_0^{\pi/2} sin\theta e^{sin\theta}$

book answer = e - 1

__6.__   $ \int_0^1 \int_0^v \sqrt{1 - v^{2}}\;dudv$

$ = \int_0^1 [u\sqrt{1 - v^{2}}]_{u=0}^{u=v}\;dv = \int_0^1 ([v\sqrt{1 - v^{2}}] - [(0)\sqrt{1 - v^{2}}]) \;dv $

$ = \int_0^1 v\sqrt{1 - v^{2}} \;dv $

Using integrations by parts: </br>
$\int u dv = uv - \int vdu $</br>
let $u =  $, and $ dv = $</br> 
then $du = $, and $v = $
</br>

__Exercises 7 to 11__ Evaluate the integrals

__7.__   $ \int \int_D x^3y^2\;dA, D = \{(x,y)| 0≤x≤2, -x≤y≤x\}$

 $ = \int_0^2 \int_{-x}^x x^3y^2\;dydx = \int_0^2 [\frac{1}{3} x^3y^3]_{y=-x}^{y=x} dx$

$ = \int_0^2 ([\frac{1}{3} x^3x^3] - [\frac{1}{3} x^3(-x)^3]) dx = \int_0^2 ([\frac{1}{3} x^6] - [-\frac{1}{3}x^6]) dx$

$ = \int_0^2 [\frac{2}{3} x^6] dx = [(\frac{1}{7})(\frac{2}{3})x^7]_0^2 = [\frac{2}{21}x^7]_0^2$

$ = [\frac{2}{21}(2)^7]-[0] = \frac{256}{21}$, this is also the book answer

__8.__   $ \int \int_D \frac{4y}{x^3+2};dA, D = \{(x,y)| 1≤x≤2, 0≤y≤2x\}$

$ = \int_1^2 \int_0^{2x} \frac{4y}{x^3+2} \;dydx = \int_1^2 \int_0^{2x} \frac{1}{x^3+2} 4y\;dydx = \int_1^2 [\frac{1}{x^3+2} \frac{4}{2} y^2]_{y=0}^{y=2x} dx$

$ = \int_1^2 ([\frac{1}{x^3+2} 2(2x)^2] - [\frac{1}{x^3+2} 2(0)^2]) dx = \int_1^2 \frac{1}{x^3+2} 8x^2\;dx $

Using u-subtitution,  

let $ u = x^3 + 2 $ and then $ du = 3x^2 $

$ = \frac{8}{3} \int_1^2 \frac{1}{x^3 + 2} 3x^2\;dy = \frac{8}{3} \int_1^2 \frac{1}{u} du $</b>

Remember, that $\int \frac{1}{x} = ln|x| + C$ 

Thus, $ \frac{8}{3} [ ln |x^3+2| ]_1^2 $



$ = \frac{8}{3} [ ln |10| - ln |3| ] $

Remembering properties of natural logs $ ln\;\frac{a}{b} = ln\;a - ln\;b$

$ = \frac{8}{3}\;ln\;|\frac{10}{3}|$

__9.__   $ \int \int_D \frac{2y}{x^2+1};dA, D = \{(x,y)| 0≤x≤1, 0≤y≤\sqrt{x}\}$

$ = \int_0^1 \int_0^{\sqrt{x}} \frac{1}{x^2+1} 2y \;dydx = \int_0^1 [\frac{1}{x^2+1} \frac{2}{2} y^2]_{y=0}^{y=\sqrt{x}} dx = \int_0^1 [\frac{1}{x^2+1} y^2]_{y=0}^{y=\sqrt{x}} dx$

$ = \int_0^1 ([\frac{1}{x^2+1} (\sqrt{x})^2]-[\frac{1}{x^2+1} (0)^2]) dx = \int_0^1 [\frac{x}{x^2+1}]dx$

$ = \int_0^1 [\frac{x}{x^2+1}]dx$

Again, using u substitution, let $ u = x^2 +1 $ and $ du = 2x $ 

$ = \int_0^1 [\frac{x}{x^2+1}]dx = \frac{1}{2} \int_0^1 \frac{1}{u} \;du = \frac{1}{2} [\;ln|u|\;]_0^1 $</b>

$ = \frac{1}{2} [\;ln |(x^2 + 1)|\;]_0^1 = \frac{1}{2} [ln|2| - ln|1|] $</br>

$ = \frac{1}{2} [ln|\frac{2}{1}|] =  \frac{1}{2} [ln|2|] $

__10.__   $ \int \int_D e^{y^2}\;dA, D = \{(x,y)| 0≤y≤1, 0≤x≤y\}$

$ = \int_0^1 \int_0^y e^{y^2} dxdy = \int_0^1 \int_0^y e^{y^2} dxdy$ 

$ = \int_0^1 [xe^{y^2}]_{x=0}^{x=y} dy = \int_0^1 [ye^{y^2}] - [(0)e^{y^2}] dy = \int_0^1 ye^{y^2} dy$ 

Given that $ \int e^{f(x)} f'(x) dx = e^{f(x)} $

$ \frac{1}{2} \int_0^1 2ye^{y^2} dy = \frac{1}{2} [e^{y^2}]_0^1 = \frac{1}{2}(e^1-e^0) = \frac{1}{2}e -\frac{1}{2} $ 

__11.__   $ \int \int_D e^{\frac{x}{y}}\;dA, D = \{(x,y)| 1≤y≤2, y≤x≤{y^3}\}$

$ = \int_1^2 \int_y^{y^3} e^{\frac{x}{y}} dxdy =  \int_1^2 y \int_y^{y^3} \frac{1}{y} e^{\frac{x}{y}} $ 

$ =  \int_1^2 y [ e^{\frac{x}{y}} ]_{x=y}^{x=y^3} dy$

$ = \int_1^2 y ( [ e^{\frac{y^3}{y}} ] - [e^{\frac{y}{y}}] )dy = \int_1^2 y([e^{y^2} ] - [e])dy $

$ = \int_1^2  (ye^{y^2} - ye)\;dy = \frac{1}{2} \int_1^2 (2ye^{y^2} - 2ye) dy$

$ =  \frac{1}{2}  [e^{y^2} - y^2e]_1^2 =  \frac{1}{2} [e^{y^2} - y^2e]_1^2 = \frac{1}{2} ([e^4 - y^2e]-[e - e])$

 $ = \frac{1}{2} [e^4 - 4e] - [ e - e] = \frac{1}{2}e^4 - 2e$

$ book answer = \frac{1}{2}e^4 - 2e $ 

__12.__   $ \int \int_D x \sqrt{y^2 - x^2} \;dA, D = \{(x,y)| 0≤y≤1,\;0≤x≤y\} $

$ = \int_0^1 \int_0^y x\sqrt{x^2-y^2} dydx =  \int_0^1 \int_0^y  (x^2-y^2)^{\frac{1}{2}} dydx $ 

Using substitution, let $ u = (y^2 - x^2), du/dx = -2x $

$ \int_0^y u^{\frac{1}{2}} \frac{du}{dx} = \frac{-1}{2} \int_0^y u^{\frac{1}{2}} \frac{du}{dx} = \frac{-1}{2} [\frac{2}{3} u^{\frac{3}{2}}]_{x=0}^{x=y} = \frac{-1}{2} [\frac{2}{3} (y^2-x^2)^{\frac{3}{2}}]_{x=0}^{x=y}$

$ = \frac{-1}{2} ( [\frac{2}{3} (y^2-y^2)^{\frac{3}{2}}] - [\frac{2}{3} (y^2-(0)^2)^{\frac{3}{2}}] ) = \frac{-1}{2} ( [0] - [\frac{2}{3} (y^2)^{\frac{3}{2}}] ) = \frac{-1}{2} ( -[\frac{2}{3} y^2(\sqrt{y^2})] ) = \frac{2}{6} y^2(y) = \frac{2}{6} y^3$

__13.__   $ \int \int_D x cos(y) \;dA, bounded\;by\;y = 0, y = x^2, x=1 $

placing this back into the original integral to complete the final integration

$ \int_0^1 \frac{2}{6} y^3 dy = [(\frac{2}{6}) (\frac{1}{4}) y^4]_0^1 = [(\frac{2}{24}) y^4]_0^1 = \frac{2}{24} $

__14.__   $ \int \int_D (x+y) \;dA, D\;is\;bounded\;by\;y = \sqrt{x}, y = x^2$

For this problem, x-limits can be found where the two functions intersect, so when $\sqrt{x} = x^2$ or $x = -1, 1$
</br> So, 
$D = \{(x,y)| -1≤y≤1, \sqrt{x}≤x≤{x^2}\} $
</br>
$\int_0^1 \int_{y=\sqrt{x}}^{y=x^2} (x+y)\;dydx $


$ = \int_0^1 ([(y)(x) + \frac{1}{2} (y^2) ]_{y=\sqrt{x}}^{y =x^2}\;dx $

$ = \int_0^1 ([(x^2)(x) + \frac{1}{2} ((x^2)^2) ] - [(\sqrt{x})(x) + \frac{1}{2} (\sqrt{x})^2 ]_{y=\sqrt{x}}^{y =x^2}\;dx $

$ = \int_0^1 ([(x^3) + \frac{1}{2} (x^4) ] - [(x^{\frac{3}{2}}) + \frac{x}{2} ] dx $

$ = \int_0^1 (\frac{1}{2}(x^4) + x^3 - x^{\frac{3}{2}} - \frac{x}{2}) dx $

$ = [(\frac{1}{10}x^5 + \frac{1}{4}x^4 - \frac{2}{5}x^{\frac{5}{2}} - \frac{1}{4}x^2)]_0^1 dx $

$ = [\frac{1}{10} + \frac{1}{4} - \frac{2}{5} - \frac{1}{4}] - [0] = [\frac{2}{20} - \frac{8}{20}] = -\frac{6}{20}$

__39-44 Evaluate the integral by reversing the order of integration__

__39.__   $ \int_0^1 \int_{3y}^{3} e^{x^2} \;dxdy $

$ = \int_{3y}^{3} \int_0^1  e^{x^2} \;dydx = \int_{3y}^{3} [ ye^{x^2}]_0^1 \;dx $

$ = \int_3^{y^3} ([(1)e^{x^2}]- [0]) \;dx  = \int_3^{y^3} e^{x^2}\;dx$ # Error function?  

$ = [ e^{(y^3)^2} ] - [ e^{3^2} ] $

book ans: $(e^9 - 1)/6$

JHUAPL is my real internship of 2003. I try to keep fresh, because there is literlly nothing else to do.  I am not going to record stories of people being nasty to me.  I am not a exective come down off high... I am an abused woman coming right out of first hell, who invested in stocks while HOMELESS.  Because $400 isn't exactly high flying.  The gangsta's paradise men see more than that a day.  Waitresses make more than that in a weekend.  I receive 0 dollars in disability benefits.  I receive 0 dollars in loans.  I receive 0 dollars in veterans benefits.  I have 0 income per month.  I live in a tent, which makes me look like first, because men decided there was nothing good about me, and I shouuld be grateful to have a ride along, even if it stymied a life of my own.  An umbrella so I can have time to find my people, is appreciated.  But telling me I cannot have a life of my own... no thanks. I am already DTX AND DXT and was never been fired from a job, other than waitressing. I am not a dog for the trans community, nor should I have to carry anti-trans books. That wasn't one of my matches - it was my intolerant hateful MOTHER who told me that I couldn't stay with her, was afraid I would turn my brother gay if his fingernails got painted.  The LGBTQ community should love me, but they don't... because if someone says my real name is anti-trans, then I should be a slave for them.  I am like the ring itself in the Hobbit movies, the power of "owning" me makes everyone keep me low to exploit me. Of course I am a freaking mess.

__40.__   $ \int_0^1 \int_{\sqrt{y}}^1 \sqrt{x^3+1} \;dxdy $

$ \int_0^1 \int_{\sqrt{y}}^1 \sqrt{x^3+1} \;dxdy  =  $

$ \int_{\sqrt{y}}^1 \int_0^1  \sqrt{x^3+1} \;dydx = \int_{\sqrt{y}}^1 [ y\sqrt{x^3+1}]_0^1 \;dx$

$ = \int_{\sqrt{y}}^1 [ (1)\sqrt{x^3+1}-[ (0)\sqrt{x^3+1}] \;dx $

$ = \int_{\sqrt{y}}^1 \sqrt{x^3+1}\; \;dx = $

let $u = x^3+1 ,  \int\sqrt{u}\;du =  \frac{2}{3} u^{\frac{3}{2}}$, this doesn't help.  


__41.__   $ \int_0^3 \int_{y^2}^9 ycos(x^2) \;dxdy $

$ = \int_{y^2}^9 \int_0^3  ycos(x^2) \;dydx = \int_{y^2}^9  [ycos(x^2)]_0^3 \;dydx = \int_{y^2}^9  [(3)cos(x^2)]-[(0)cos(x^2)]\;dx = ycos(x^2)]_0^3 \;dydx $

$ = \int_{y^2}^9 3cos(x^2) dx$</br>
Again, reverse chain rule does not work because the derivative of $cos(f(x))$ is not present.</br>

I am human, and flawed.  But people would absolutely crucify Jesus again. I am not christian, but I believe he was a man that WAS crucified and did not deserve crucifixion and yet most christians know nothing of that.  Oh no, a god, he is perfect of course we would recognize him... we wouldn't nail him to a cross?  Oh, really? Not even if Pontius Pilate said he was bad?  Are you so sure you wouldn't judge someone on what someone else TOLD you was true? 

Just like you believed that I was addicted to drugs or killed people or was everything under the sun... 
I am your intel student, who hasn't done any of the horrible things you made me pay for because I was sarcastic and said "I eat babies with Wasabi" or claimed you treated me like a terrorist.  You DID. You h


This is a Ginormous law enforcement steal.  They beat the shit out of me, and you'll make me a slave to them. They are NOT funding me.  They NEVER did. Individuals may have, but not them.  I didn't fail FlatIron, I didn't have a place to stay and pawned the laptop.  I mean I am not a developer anyway, but eveything you build dies, because no one funds ME to do anythying.  And yes, I am allowed to ask for an opportunity to do what normal people do.  You dont' even let me wash dishes like I did on the east coast?  You can't make me an undocumented worked, when I dont' have another language or ID.  You did not develop me at all.  I developed myself under umbrellas.

__The most powerful way to attack a book is to demonize its author, to turn him into a creature of base motives and evil intentions.  The "Satan Rushdy" who would afterward be paraded down the world's streets by angry demonstrators, hanged in effigy with a red tongue hanging out and wearing a crude tuxedo was being created.  Here was the first proposition of the assault: that anyone who wrote a book with the word "satanic" in the title must be satanic too.  Like many false propositions that flourished in the incipient age of information (or disinformation), it became true by repitition.  Tell a lie about a man once and many people will not believe you.  Tell it a million times and it is the man himself who will not be believed. SR, Joseph Anton p.112

Remembering Integration by Parts:
$ \int f(x)g'(x) = f(x)g(x) - \int f'(x)g(x)dx $

Let $f(x) = $ and $g'(x) = $, then $ f'(x) =  $ and $ g(x) =  $ 